superset/assets/javascripts/explorev2/reducers/exploreReducer.js

```diff
@@ -90,6 +87,7 @@ export const exploreReducer = function (state, action) {
           chartUpdateEndTime: null,
           chartUpdateStartTime: now(),
           triggerQuery: false,
+          queryRequest: action.queryRequest,
```

Comment:
i think  @mistercrunch's suggestion of keeping track of `latestQueryId` in state might be a good solution here. so we wouldn't keep track of the xhr request and would not call `xhr.abort()` but we would keep track of `latestQueryId` and only handle the results when the request returns if the `latestQueryId` and id in the response match. this way we won't have to put a function into our redux store, keeping with redux best practices.